__tests__/server/utils/pollModuleMap.spec.js

```diff
@@ -328,7 +323,7 @@ describe('pollModuleMap', () => {
     // check that the poll interval is reset to min.
     expect(setTimeout).toHaveBeenCalledWith(pollModuleMap, MIN_POLL_TIME);
 
-    expect(setGauge).toHaveBeenCalledTimes(1);
+    expect(setGauge).toHaveBeenCalledTimes(3);
     expect(setGauge).toHaveBeenCalledWith(holocronMetrics.moduleMapPollWait, 5);
```

Comment:
should this include
```suggestion
    expect(setGauge).toHaveBeenCalledWith(holocronMetrics.moduleMapPollWait, 5);
    expect(setGauge).toHaveBeenCalledWith(holocronMetrics.rejectedModules, Infinity);
    expect(setGauge).toHaveBeenCalledWith(holocronMetrics.modulesRequiringFallbacks, Infinity);
```
alongside the `.toHaveBeenCalledTimes(` assertion updates?

Reply:
added below

Reply:
Do we need this in all the tests looking at the telemeters? 🤔

Reply:
each telemeter has a specific test already see https://github.com/americanexpress/one-app/pull/1131/files#diff-82cec72196769195fc18f858b882683ecae08a9f5fc3b7d097fae4f60d33b404R330-R331
which is why i didn't include the other metrics in this test